### ISSUE TITLE
Packaging arbitrary fails

### ISSUE THREAD

**Issue by kvnwolf** [serverless/serverless-babel-plugin#14]
When I try to run `sls deploy`, sometimes it throws errors on the `Packaging service with compiled files..` step, this is the output:

```
Error --------------------------------------------------

     Error: Cannot find module './isFunction' (While processing
     preset: "/Users/kevinwolf/Projects/Playground/serverless-test/.serverless/tmpBabelDirectory/node_modules/babel-preset-es2015/lib/index.js")

        at Function.Module._resolveFilename (module.js:472:15)

        at Function.Module._load (module.js:420:25)
    at
     Module.require (module.js:500:17)
    at require (internal/module.js:20:19)

        at Object.<anonymous> (/Users/kevinwolf/Projects/Playground/serverless-test/.serverless/tmpBabelDirectory/node_modules/lodash/_baseIsNative.js:1:80)

        at Module._compile (module.js:573:32)
    at Object.Module._extensions..js
     (module.js:582:10)
    at Module.load (module.js:490:32)

        at tryModuleLoad (module.js:449:12)
    at Function.Module._load
     (module.js:441:3)


     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues

     Please report this error. We think it might be a bug.

  Your Environment Information -----------------------------
     OS:                 darwin
     Node Version:       7.1.0
     Serverless Version: 1.8.0
```

This does not happen all the time, it appears to be happening randomly.
Any thoughts?

**Comment by duncanfinney** [comment]
Seeing the same thing. It seems that the plugin isn't waiting for the file copy to finish before moving on:

![image](https://cloud.githubusercontent.com/assets/3267774/24375780/896b69f8-1307-11e7-9dca-362a63f264fb.png)

**Comment by nikgraf** [comment]
@duncanfinney just published 0.2.0. Can you try again?

**Comment by duncanfinney** [comment]
@nikgraf I was just going through random things and realized I never replied to this. It works now. 

I'm guessing 0.2.0 was the fix (5 months ago)!
